example/lib/main.dart

```diff
@@ -47,12 +43,64 @@ class _MyAppState extends State<MyApp> {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      debugShowCheckedModeBanner: false,
       home: Scaffold(
         appBar: AppBar(
-          title: const Text('Plugin example app'),
+          title: const Text(
+            ExampleStrings.appBarTitle,
+          ),
         ),
         body: Center(
-          child: Text('Running on: $_platformVersion\n'),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              Column(
+                children: [
+                  PluginButton(
+                    text: ExampleStrings.checkPermission,
+                    onPressed: () {
+                      AndroidLocatorPlugin.checkPermission;
+                    },
+                  ),
+                  StreamBuilder(
+                    stream: AndroidLocatorPlugin.accessStream,
+                    builder: (
+                      BuildContext context,
+                      AsyncSnapshot<dynamic> snapshot,
+                    ) {
+                      return snapshot.hasData
+                          ? Text(
+                              snapshot.data.toString(),
+                            )
+                          : Text(
+                              ExampleStrings.permissionInfo,
+                            );
+                    },
+                  ),
+                ],
+              ),
+              Column(
+                children: [
+                  PluginButton(
+                    text: ExampleStrings.requestPermission,
+                    onPressed: () {
+                      AndroidLocatorPlugin.requestPermission;
+                    },
+                  ),
+                ],
+              ),
+              Column(
+                children: [
+                  PluginButton(
+                    text: ExampleStrings.initializePlugin,
+                    onPressed: () {
+                      AndroidLocatorPlugin.initializePlugin;
+                    },
+                  ),
+                ],
```

Comment:
Why you need to create two columns? Do you think that is better to has only one column, like this:
Column(
     children: [
           PluginButton( ... ),
           PluginButton( ... ),
          .....
     ],
)

Reply:
You can wrap the button with a padding if you don't want to be to closed

Reply:
Also don't use Column with only one child, it loses the purpose of the Columns,Rows, Widgets that support more elements inside

Reply:
Ready. Only one column is now implemented with padding in their buttons